config/jetson-node-k2-front.json

```diff
@@ -63,11 +63,11 @@
           "init": {
             "serial_number": "939622072545",
             "device": "D400",
-            "depth_subsample": 1,
+            "depth_subsample": 5,
             "depth_rgb": true,
             "depth_resolution": [640, 480],
             "rgb_resolution": [640, 480],
-            "depth_fps": 6
+            "depth_fps": 30
```

Comment:
There is a risk that this reduces latency, as desired, but then the 200ms timestamp offset gets too big.